src/JupyterLibrary/clients/jupyterlab/Output.robot

```diff
@@ -39,3 +39,16 @@ Screenshot Each Output of Active JupyterLab Document
         Screenshot Each Output of Active JupyterLab Cell    ${prefix}_cell_${i}
         Screenshot Markdown of Active JupyterLab Cell    ${prefix}_cell_${i}
     END
+
+Wait Until a Given JupyterLab Code Cell Is Not Active In a Given Tab
+  [Documentation]  Waits until the given cell in a specific JL tab no longer has an active prompt "[*]:".
+  ...              The cell index starts from 1 and you must count empty and markdown cells too.
+  [Arguments]  ${tab_id_to_wait}  ${cell_n}  ${timeout}=120seconds
+  Wait Until Element Is Not Visible  xpath=//div[@aria-labelledby="${tab_id_to_wait}"]/${NB TAB XP CONTENT}/div[${cell_n}]/${CELL XP INPUT STATUS ICON}   ${timeout}
+
+Get A JupyterLab Code Cell Output In a Given Tab
+   [Documentation]  It returns the output of a given cell in a specific JL tab.
+   ...              The cell index starts from 1 and you must count empty and markdown cells too.
+   [Arguments]  ${tab_id_to_read}  ${cell_n}
```

Comment:
This one's tough. Also there can be many outputs, which many be worth parameterizing. Anyhow, I'd probably:
- get a handle on the cell
- get a handle on the output
- then use SL's `Get Text` on the element

Reply:
I didn't consider other output type (e.g., images) other than text. So, to write a more generic keyword, I thought to return all the output child elements. I could include a parameter to fetch one specific cell by its progressive number. After that, I could write a keyword to extract the text or the image (i'm not sure if there are more than two output types for jupyter)